Source/Scripts/carthage-zsh-completion

```diff
@@ -48,6 +48,7 @@ _carthage() {
                         '--no-build' \
                         '--color: :(auto always never)' \
                         '--verbose' \
+                        '--project-directory: :_files' \
```

Comment:
It might be better to use `_directories` for `--project-directory` if it accepts only a directory.

```
'--project-directory: :_directories' \
```